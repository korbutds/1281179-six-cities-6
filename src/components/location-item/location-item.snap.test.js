import React from 'react';
import {render} from '@testing-library/react';
import {Test} from '../../test/test-mock';
import LocationItem from './location-item';
test(`Location Item component render correctly`, () => {
  const {container} = render(
      <Test>
        <LocationItem onClickHandle={jest.fn()} city={`Amsterdam`} />
      </Test>
  );
  expect(container).toMatchSnapshot();
});
