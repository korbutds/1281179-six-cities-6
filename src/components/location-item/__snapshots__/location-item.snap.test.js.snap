// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Location Item component render correctly 1`] = `
<div>
  <li
    class="locations__item"
  >
    <a
      class="locations__item-link tabs__item "
      href="#"
    >
      <span>
        Amsterdam
      </span>
    </a>
  </li>
</div>
`;
